### PR TITLE
NAP-189 Add sub-type for passenger-transportation type

### PR DESCRIPTION
**Pull request by rckopteri**

### CODE REVIEW

File: ote/src/cljs/ote/app/controller/transport_service.cljs [hunk 1]
```diff
@@ -14,7 +14,32 @@
...
+  "Returns the service column keyword for the given type enum value."
+  [type]
+  (case type
+    :passenger-transportation-taxi :taxi
+    :passenger-transportation-request :request
+    :passenger-transportation-schedule :schedule
+    :terminal :terminal ;; No subtype for terminals - but it is still saved to database
+    :rentals :rentals
+    :parking :parking
+    :brokerage :brokerage))
```
Comment (tatut) [inline ote/src/cljs/ote/app/controller/transport_service.cljs]: change names of  `convert-subtype-type-to-service-type-key` and `get-subtype-from-subtype-key` to something simpler and remove verbiage

File: ote/src/cljs/ote/views/passenger_transportation.cljs [hunk 2]
```diff
@@ -45,7 +45,13 @@
     :layout :row}
 
    {:name ::t-service/name
-    :type :string}))
+    :type :string}
+
+   {:style {:width "350px"}
```
Comment (tatut) [inline ote/src/cljs/ote/views/passenger_transportation.cljs]: hard coded pixel width should be moved to a style ns

File: ote/src/cljs/ote/app/controller/transport_service.cljs (hunk 1 above)
Comment (rckopteri) [reply ote/src/cljs/ote/app/controller/transport_service.cljs]: Changed

File: ote/src/cljs/ote/views/passenger_transportation.cljs (hunk 2 above)
Comment (rckopteri) [reply ote/src/cljs/ote/views/passenger_transportation.cljs]: Fixed

**Pull request merged**